App.js

```diff
@@ -11,9 +11,12 @@ import { FindScreen } from './screens/FindGamePage';
 import { WaitingScreen } from './screens/WaitingPage';
 import { FoundScreen } from './screens/FoundGamePage';
 import { FoundAPartnerScreen } from './screens/FoundAPartner';
-
+//add something to home page about Dart pong
```

Comment:
Remove these comments. You can create tasks so we have it on the ZenHub board. If you do move it to the board, make sure to add it to the correct milestone and estimate value